_opensearch/supported-field-types/xy-point.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: default
+title: xy point
+nav_order: 58
+has_children: false
+parent: Cartesian field types
+grand_parent: Supported field types
+---
+
+# xy point field type
+
+An xy point field type contains a point in a two-dimensional Cartesian coordinate system, specified by x and y coordinates. It is based on the Lucene [XYPoint](https://lucene.apache.org/core/8_5_1/core/org/apache/lucene/geo/XYPoint.html) field type. The xy point field type is similar to the [geopoint]({{site.url}}{{site.baseurl}}/opensearch/supported-field-types/geo-point/) field type, but does not have the range limitations of geopoint. The coordinates of an xy point are single-precision floating point values. For information about the range and precision of floating-point values, see [Numeric field types]({{site.url}}{{site.baseurl}}/opensearch/supported-field-types/numeric/).
```

Comment:
The link added for lucene is from 8.5 release. Just want to know do we update our links in the documentation to point to the latest version of lucene?

Reply:
I updated the link to 9.3.0, which is the version of Lucene that OpenSearch uses now. We have not been updating Lucene links in the docs, but we can add this step as part of our release process.